app/views/content/terms_candidate.html.erb

```diff
@@ -63,9 +63,9 @@
 
     <h3 class="govuk-heading-s">Making changes to your application</h3>
 
-    <p class="govuk-body">Once you submit an application, it is sent to the teacher training provider you applied to.</p>
+    <p class="govuk-body">When you submit your first application, you will have 5 working days to make any changes to all the sections in the tab called ‘Your details’.</p>
 
-    <p class="govuk-body">You will be able to make changes to these details after you submit an application:</p>
+    <p class="govuk-body">After 5 working days from submitting your first application, you will only be able to make changes to:</p>
```

Comment:
It might be worth interpolating the value `5` in case it changes to 3 or 10 in the future

Reply:
> It might be worth interpolating the value `5` in case it changes to 3 or 10 in the future

How do I do this like a professional dev? Or is this something only a dev can do (a professional one).

Reply:
`5 working days` is used in a lot of our views. Not sure it's worth doing this change here.